superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -169,13 +182,14 @@ function ListViewCard({
   imgPosition = 'top',
   cover,
 }: CardProps) {
+  const Link = url && linkComponent ? linkComponent : AnchorLink;
```

Comment:
Is this just in case this is used outside of a react-router context?

Reply:
Most list view links still need to be anchor links. It's just the dashboard ones that can be `<Link>`s for now.